src/utils/bisect.js

```diff
@@ -21,23 +21,30 @@
     THE SOFTWARE. 
 */
 
+// @flow
+
 /**
  * Calculates the index of the Array where item X should be placed, assuming the Array is sorted.
  *
```

Comment:
Do you think you can add a comment at the top of this file, explaining the difference between both functions? You can steal some of my explanation from the issue :-)